easybuild/easyblocks/generic/cmakemake.py

```diff
@@ -143,6 +143,10 @@ def configure_step(self, srcdir=None, builddir=None):
 
         options = ['-DCMAKE_INSTALL_PREFIX=%s' % self.installdir]
 
+        if self.installdir.startswith('/opt'):
+            # https://cmake.org/cmake/help/latest/module/GNUInstallDirs.html
+            options.append("-DCMAKE_INSTALL_SYSCONFDIR=%s/etc" % self.installdir)
```

Comment:
w.r.t. to `CMAKE_INSTALL_FULL_<dir>`, it's generated from `CMAKE_INSTALL_<dir>` and it will only be different from it if this value is not an absolute path, and we're setting it to an absolute path, so it's correct, no?

Reply:
Yes, I got a bit confused by the wording, should be OK.

Reply:
w.r.t. the rest, see https://github.com/easybuilders/easybuild-easyblocks/pull/2124